### ISSUE TITLE
[BUG] Chat response not showing up in UI

### ISSUE THREAD

**Issue by bijucyborg** [h2oai/h2o-llmstudio#196]
### 🐛 Bug

The response to the chat is not showing up in the chat tab. My question is displayed but not the response.
I can see the response generated in the application log but not on the UI.

### To Reproduce

I'm running H2O LLM studio v0.0.4-dev on Ubuntu 22.04.2 LTS Python 3.10.6. Am running this on a remote server and using SSH tunnelling to access port 10101.

All the steps go though fine, but when I load the fine tuned model and try to chat, the response is not displayed in the chat window.

Here is what I see in the logs
`2023-06-28 09:00:57,019 - INFO: Full prompt: <|prompt|>what info do you have?<|endoftext|><|answer|>
2023/06/28 09:01:01 * /9c28ee80-ee72-46d7-8e65-1612940f773e {"d":[{"k":"experiment/display/chat data 011","v":["I am a language model trained on the MNIST dataset, which is an image recognition task. I can provide more information if requested.\n\n1) What type of data are you using for training your model? Is it raw images or text files? If so, what kind of preprocessing and feature extraction techniques did you use to extract features from them?\n2) How many layers were used in your network architecture? Did they all share weights between each other? Are there any shared parameters that could be optimized during training?\n3) Do you train your model with stochastic gradient descent (SGD)? Or does it rely heavily on momentum?\n4) Have you tried different optimizers such as Adam or RMSprop? Which one worked best for you?\n5",false]}]}
2023-06-28 09:01:01,244 - INFO: HTTP Request: PATCH http://127.0.0.1:10101/9c28ee80-ee72-46d7-8e65-1612940f773e "HTTP/1.1 200 OK"`

And here is what I see on the UI.

<img width="942" alt="Screenshot 2023-06-28 at 11 07 02" src="https://github.com/h2oai/h2o-llmstudio/assets/24872459/60c81279-027e-457a-babb-9c542117ff9a">

Here are the logs
[logs_nocodellm.zip](https://github.com/h2oai/h2o-llmstudio/files/11892346/logs_nocodellm.zip)

Any suggestions on what might be wrong with the setup?

**Comment by psinger** [comment]
@bijucyborg please pull master, reinstall packages, and retry, should be fixed now

**Comment by tmm1** [comment]
same issue here on the docker nightly, i presume it hasn't been rebuilt since the fix on master
